control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -2207,6 +2207,10 @@ components:
           description: the drain state
           allOf:
             - $ref: '#/components/schemas/CordonDrainState'
+        node_nqn:
+          description: hosts allowed to connect the target
+          allOf:
```

Comment:
why do you need allOf here?

Reply:
Not needed yes. Has only 1 schema. Removed allOf